docs/about-ensembl/the-ensembl-project/cite.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: cite
+title: Citing Ensembl
+description:
+status: draft
+---
+
+# Citing Ensembl
+
+If you've used Ensembl in your work, please cite the appropriate overview article:
+
+## For vertebrate genomes
+
+*Ensembl 2020*
+_Nucleic Acids Res._ 2021, vol. 49(1):884–891
+[doi:10.1093/nar/gkaa942](https://doi.org/10.1093/nar/gkaa942)
+
+## For invertebrate, plant, fungal and prokaryote genomes
+
+*Ensembl Genomes 2020&#2014;enabling non-vertebrate genomic research*
```

Comment:
What's this after 2020?

Reply:
It's meant to be an m-dash, as per the correct punctuation of the article title - I googled how to do them in Markdown and it was suggested that numerical entities were the way to go.

Reply:
If it proves problematic, we can replace it with two hyphens or a hyphen with a space either side.

Reply:
Option-Shift-minus should do it on a Mac (—).

Reply:
Maybe I need to do it in an IDE - it looks like a hyphen in vim 😞